### DISCUSSION TITLE
How to run on 0.0.0.0

### DISCUSSION THREAD

**Discussion by richardji7** [nocodb/nocodb#5290] [General]
Hi All,

A bit at lost here.  Whether I run the binary version or the Node.js, it always starts on localhost:8080.  How do I make it to listen on 0.0.0.0 so I can hit it from another computer since I am running it headless?

I tried to add to .env the following but it didn't work:
NUXT_PUBLIC_NC_BACKEND_URL=http://0.0.0.0:8080

Appreciate the guidance.

**Comment by o1lab** [comment]
You dont've to set the URL. 

Instead you have the know the IP of your server/system available and try accessing it.

**Reply by richardji7** [reply]
Ah, thanks.  When I saw localhost, I assumed it just listens on localhost and since I didn't open firewall port, it didn't work.  I opened the port 8080 and it worked!
